### PR TITLE
nowarp mode and new online coadds location

### PR DESCRIPTION
**Pull request by esheldon**
Mode where there is no warping

### CODE REVIEW

**Comment by esheldon** [comment]
this seems to be working fine in full sims

**Comment by esheldon** [comment]
piggybacking work on issue #57

**Comment by esheldon** [comment]
for some reason the stackvana that gets installed isn't the latest

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -11,7 +11,9 @@ jobs:
       name: tests
       strategy:
           matrix:
-            pyver: [3.8]
+            # specific version 3.8.8 is here to fix solver error w/ sqlite 
+            # see https://lsstc.slack.com/archives/C2K9RHYTV/p1628567486002200
```
Comment (beckermr) [inline .github/workflows/test.yaml:15]:
```suggestion
            # see https://lsstc.slack.com/archives/C2K9RHYTV/p1628567486002200
            # when python 3.9 is in use, set this to 3.9
```

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -31,7 +34,7 @@ jobs:
                 conda config --set always_yes yes
                 conda install -q mamba
 
-                mamba install -q stackvana=0
+                mamba install -q stackvana=0.2021.32
```
Comment (beckermr) [inline .github/workflows/test.yaml:37]:
```suggestion
                mamba install -q stackvana
```

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -31,7 +34,7 @@ jobs:
                 conda config --set always_yes yes
                 conda install -q mamba
 
-                mamba install -q stackvana=0
+                mamba install -q stackvana
```
Comment (beckermr) [inline .github/workflows/test.yaml:37]:
```suggestion
                mamba install -q stackvana=0
```

**Review by beckermr — APPROVED** [review]

**Comment by esheldon** [comment]
thanks for the CI help!

**Pull request merged**